File: maiworld-seller-web/src/main/webapp/js/service/evaluationService.js
//服务层
		app.service('evaluationService',function($http){
			
			this.findAll=function(){
				return $http.get('../evaluation/findAll.do');
			}
			
			this.findPage=function(page,rows){
				return $http.get('../evaluation/findPage.do?page='+page+'&rows='+rows);
			}
			
			this.findOne=function(id){
				return $http.get('../evaluation/findOne.do?id='+id);
			}
			
			this.add=function(entity){
				return $http.post('../evaluation/add.do', entity);
			}
			
			this.update=function(entity){
				return $http.post('../evaluation/update.do', entity);
			}
			
			this.dele=function(ids){
				return $http.get('../evaluation/delete.do?ids='+ids);
			}
			this.search=function(page, rows, searchEntity){
				return $http.post('../evaluation/evaluationList.do?page='+page+'&rows='+rows, searchEntity);
			}
			
			this.evaluationList=function(){
				return $http.get('../evaluation/evaluationList.do')
			}
			
			
		});

File: maiworld-seller-interface/src/main/java/com/maiworld/seller/service/EvaluationService.java
package com.maiworld.seller.service;

import java.util.List;

import com.maiworld.pojo.TbEvaluation;

import entity.PageResult;

/**
 * 评论表接口
 * @author Administrator
 *
 */
public interface EvaluationService {

	
	public List<TbEvaluation> findAll();
	
	public PageResult findPage(int pageNum, int pageSize);
	
	public void add(TbEvaluation evaluation);
	
	public void update(TbEvaluation evaluation);
	
	public TbEvaluation findOne(Long id);
	
	public void delete(Long[] ids);
	
	public PageResult findPage(TbEvaluation evaluation, int pageNum, int pageSize);
	
	public PageResult evaluationList(int pageNum, int pageSize );
	
}
